legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/state/ExecutionState.java

```diff
@@ -126,9 +126,17 @@ public ExecutionState varName(String var)
     }
 
     public ExecutionState setAuthUser(String user)
+    {
+        return setAuthUser(user, true);
+    }
+
+    public ExecutionState setAuthUser(String user, boolean setTransformAllocation)
     {
         this.authId = user;
-        this.transformAllocation = true;
+        if (setTransformAllocation)
```

Comment:
This if check is redundant.  this.transformAllocation  = setTransformAllocation is sufficient

Reply:
Fixed. Thanks!